### PR TITLE
[CLA] Corporate signature for Uvid d.o.o.

### PR DESCRIPTION
**Pull request by uvid-m**

### CODE REVIEW

**Comment by mart-e** [comment]
Hello,

Thank you for the signature, it will soon be integrated into Odoo.
Future pull requests made on odoo will be validated by the runbot.
If you have existing PRs that are awaiting for CLA validation, don't forget to rebase your code as soon as this is merged.

Regards

@robodoo review+

**Comment by mart-e** [comment]
@uvid-m this was not merged because you pushed in the branch before it was integrated.
The CLA part is fine for me but I let @jco-odoo review the new commit about l10n_hr.

File: addons/l10n_hr/__manifest__.py [hunk 1]
```diff
@@ -34,11 +40,10 @@
 Osnovne fiskalne pozicije
 
 Izvori podataka:
- https://www.rrif.hr/dok/preuzimanje/rrif-rp2011.rar
- https://www.rrif.hr/dok/preuzimanje/rrif-rp2012.rar
+https://www.rrif.hr/dok/preuzimanje/RRIF-RP2021.PDF
 
 """,
-    "version": "13.0",
+    "version": "15.0",
```
Comment (jco-odoo) [inline addons/l10n_hr/__manifest__.py:46]: no need

File: addons/l10n_hr/data/account_tax_fiscal_position_data.xml [hunk 2]
```diff
@@ -5,11 +5,6 @@
 			<field name="name">R1 partneri</field>
 		</record>
 
-		<record id="account_fiscal_position_template_rrif_2" model="account.fiscal.position.template">
-			<field name="chart_template_id" ref="l10n_hr_chart_template_rrif"/>
-			<field name="name">R2 partneri</field>
```
Comment (jco-odoo) [inline addons/l10n_hr/data/account_tax_fiscal_position_data.xml:10]: It would be good to have some context here.

**Review by jco-odoo — COMMENTED** [review]
Nice to have an update of the chart of accounts.  It would be good to use the same format for the accounts csv, so we have a better idea of what changed exactly.  @chklop

**Pull request closed without merging**